import-export-cli/integration/README.md

```diff
@@ -0,0 +1,97 @@
+# APICTL Integration testing
+
+## Pre-requisites for running integration tests
+1. In order to run the integration tests, two instances of API Manager need to be started. These two instances will act as the APIM environments that will be used for doing exporting and importing scenarios respectively. The integration tests will execute apictl commands, DCR calls and REST API calls against these instances.
```

Comment:
Shall we mention the two instances are running with port offset 0 and 1 here as we have mentioned that is the default in the step 3.

Reply:
Fixed in https://github.com/wso2/product-apim-tooling/pull/310/commits/18287b957368281440c9d4da82ebaf51636407e2